.github/workflows/pr-go-mod-tidy-mocks.yml

```
@@ -42,6 +42,8 @@ jobs:
         with:
           go-version: "1.23"
           check-latest: true
+          # FIXME: Remove this retry after https://github.com/uber-go/mock/issues/226 get fixed
+          max_attempts: 3
```

actionlint (1.7.4)

input "max_attempts" is not defined in action "actions/setup-go@v5". available inputs are "architecture", "cache", "cache-dependency-path", "check-latest", "go-version", "go-version-file", "token"

(action)